overlay/files-ubuntu-arm-rpi/etc/elemental/config.yaml

```diff
@@ -0,0 +1,22 @@
+cosign: false
+verify: false
+install:
+  grub-entry-name: "kairos"
+  system:
+    size: 2000
+  recovery-system:
+    size: 2000
+upgrade:
+  grub-entry-name: "kairos"
+  recovery-system:
+    size: 2000
+  system:
+    size: 2000
+reset:
+  grub-entry-name: "kairos"
+  system:
+    size: 2000
+
+cloud-init-paths:
+- /run/initramfs/cos-state
+# - /run/initramfs/live
```

Comment:
remove this line if it's commented out?

Reply:
same like above, i hoped the original author left it there as a hint ;)

Reply:
fixed